lib/db.js

```diff
@@ -418,20 +458,35 @@ module.exports = (
   }
 
   DB.prototype.updateSessionToken = function (token, userAgentString) {
-    log.trace({ op: 'DB.updateSessionToken', uid: token && token.uid })
-
-    if (! token.update(userAgentString)) {
+    log.trace({ op: 'REDIS.updateSessionToken', uid: token && token.uid })
+    if (! token.update(userAgentString) || ! this.redis) {
       return P.resolve()
     }
 
-    return this.pool.post('/sessionToken/' + token.id + '/update', {
+    const uid = token.uid
+    const newToken = [{
+      tokenId: token.id,
+      uid: uid,
       uaBrowser: token.uaBrowser,
       uaBrowserVersion: token.uaBrowserVersion,
       uaOS: token.uaOS,
       uaOSVersion: token.uaOSVersion,
       uaDeviceType: token.uaDeviceType,
-      lastAccessTime: token.lastAccessTime
-    })
+      lastAccessTime: token.lastAccessTime,
+      createdAt: token.createdAt
+    }]
+    let value = []
```

Comment:
I'm well into bike-shedding territory here, but `value` seems like a slightly misleading name for the result array here. `values` or `results` or `sessionTokens` are all better names I reckon.